docs/installation/installation.rst

```diff
@@ -330,7 +363,7 @@ either of the following methods:
          <h2>Installing Docker on Windows</h2>
       
       1. Download the :program:`Docker Desktop` installer for Windows from 
-         `here <https://desktop.docker.com/win/main/amd64/Docker%20Desktop%20Installer.exe>`_ 
+         `here <https://desktop.docker.com/win/main/amd64/Docker%20Desktop%20Installer.exe>`__ 
```

Comment:
This same error happened to me before where something (I think GitHub) added a double underscore at the end of the links.